### ISSUE TITLE
Calculation Installed Capacity Seasonal Storage

### ISSUE THREAD

**Issue by matthiassulzer** [architecture-building-systems/CityEnergyAnalyst#2707]
Installed Capacities for sys_42_0 - MBS Case Study HQ
 seasonal storage with 50MW capacity is not feasible 
![installed capacity](https://user-images.githubusercontent.com/54287523/80082184-4c2d3680-8554-11ea-96df-617b7bd71fb9.png)

**Comment by shanshanhsieh** [comment]
@matthiassulzer, how about results from other individual? are they all in the similar range?

**Comment by matthiassulzer** [comment]
Jep, see plot of the 80_13 multi criteria solution
![newplot](https://user-images.githubusercontent.com/54287523/80575302-a66e4180-8a03-11ea-88ce-9dc8c6041b56.png)

**Comment by shanshanhsieh** [comment]
yes, i agree the size is a bit unrealistic as a large scale ATES is normally 10MW.
i can see these solutions:
- allow users to define the upper bounds of seasonal storage
- update the potential script to set feasible bounds
- update the bounds on the cost database
@JIMENOFONSECA, your comments will be appreciated : )

**Comment by matthiassulzer** [comment]
Dear Shanshan

Thanks for your suggestions. Seasonal storages are usually sized with the following parameters:
- Storage capacity (kWh)
- Peak load (kW)

Both parameter can be calculated (there are plenty of models available) by knowing the footprint and the depth of the geothermal storage.

Best,

Matthias

Von: Shanshan Hsieh <notifications@github.com>
Gesendet: Donnerstag, 30. April 2020 09:45
An: architecture-building-systems/CityEnergyAnalyst <CityEnergyAnalyst@noreply.github.com>
Cc: Sulzer, Matthias <Matthias.Sulzer@empa.ch>; Mention <mention@noreply.github.com>
Betreff: Re: [architecture-building-systems/CityEnergyAnalyst] Calculation Installed Capacity Seasonal Storage (#2707)


yes, i agree the size is a bit unrealistic as a large scale ATES is normally 10MW.
i can see these solutions:

  *   allow users to define the upper bounds of seasonal storage
  *   update the potential script to set feasible bounds
  *   update the bounds on the cost database
@JIMENOFONSECA<https://github.com/JIMENOFONSECA>, your comments will be appreciated : )

**Comment by JIMENOFONSECA** [comment]
Well...Shanshan you could build a new model which defines the size of the seasonal storage as a decision variable for the genetic algorithm.

Check out how we do it for the daily thermal storage for the cooling case. That seems to work well.

I advise to size the seasonal storage as a fraction of the solar potential and waste heat (in case industry, data centers, sewage heat is considerably high). Maybe you can set it to Store between 0% - 70% of the total potential. And let the algorithm just move across those limits inside the code.

Best J

Dr. Jimeno Fonseca
Senior Researcher
SEC | ETH Zurich

Read more about City Energy Dynamics<https://www.jimenofonseca.com/>

On 30 Apr 2020, 16:36 +0800, Matthias Sulzer <notifications@github.com>, wrote:

Dear Shanshan

Thanks for your suggestions. Seasonal storages are usually sized with the following parameters:
- Storage capacity (kWh)
- Peak load (kW)

Both parameter can be calculated (there are plenty of models available) by knowing the footprint and the depth of the geothermal storage.

Best,

Matthias

Von: Shanshan Hsieh <notifications@github.com>
Gesendet: Donnerstag, 30. April 2020 09:45
An: architecture-building-systems/CityEnergyAnalyst <CityEnergyAnalyst@noreply.github.com>
Cc: Sulzer, Matthias <Matthias.Sulzer@empa.ch>; Mention <mention@noreply.github.com>
Betreff: Re: [architecture-building-systems/CityEnergyAnalyst] Calculation Installed Capacity Seasonal Storage (#2707)


yes, i agree the size is a bit unrealistic as a large scale ATES is normally 10MW.
i can see these solutions:

* allow users to define the upper bounds of seasonal storage
* update the potential script to set feasible bounds
* update the bounds on the cost database
@JIMENOFONSECA<https://github.com/JIMENOFONSECA>, your comments will be appreciated : )
